### ISSUE TITLE
Redistribute error constants values

### ISSUE THREAD

**Issue by anakimluke** [godotengine/godot#9582]
On the global scope, the error constants start with `OK = 0` and then, for each next item, the value increases `1`; so you have:
```
OK = 0
FAILED = 1
ERR_UNAVAILABLE = 2
ERR_UNCONFIGURED = 3
ERR_UNAUTHORIZED = 4
...
```
But it's not like this all the way to the end, there are some breaks, like this:

```
ERR_LOCKED = 23
ERR_TIMEOUT = 24
ERR_CANT_AQUIRE_RESOURCE = 28
ERR_INVALID_DATA = 30
ERR_INVALID_PARAMETER = 31
```

**Comment by anakimluke** [comment]
I forgot to comment that though this looks broken, it may have some meaning I don't know about. Does it?

**Comment by akien-mga** [comment]
That's because not all errors defined in https://github.com/godotengine/godot/blob/df61dc4b2bd54a5a40c515493c76f5a458e5b541/core/error_list.h#L40 are exposed to GDScript in https://github.com/godotengine/godot/blob/5b3709d3096df737b8bb2344446be818b0389bfe/core/global_constants.cpp#L401

Why, I'm not sure.

**Comment by Calinou** [comment]
This appears to be solved as of 3.1, closing.
